Occasional failure to migrate ietfdb/datatracker SVN

https://github.com/ietf-ribose/svn-github-convert/runs/4677194127?check_suite_focus=true
```
2022-01-01T03:31:02.2298693Z Progress: [=============================                     ] 59.40% -
2022-01-01T03:31:02.4351155Z Progress: [=============================                     ] 59.40% \
2022-01-01T03:33:12.6435647Z svn: E170013: Unable to connect to a repository at URL 'https://svn.ietf.org/svn/tools/ietfdb'
2022-01-01T03:33:12.6436747Z svn: E000110: Error running context: Connection timed out
2022-01-01T03:33:12.6454976Z /var/lib/gems/2.7.0/gems/tractive-1.0.11/lib/tractive/revmap_generator.rb:94:in `load': invalid format, document not terminated at line 3, column 3 [parse.c:555] (Ox::ParseError)
2022-01-01T03:33:12.6456874Z 	from /var/lib/gems/2.7.0/gems/tractive-1.0.11/lib/tractive/revmap_generator.rb:94:in `commit_message_from_svn'
2022-01-01T03:33:12.6458417Z 	from /var/lib/gems/2.7.0/gems/tractive-1.0.11/lib/tractive/revmap_generator.rb:63:in `print_revmap_info'
2022-01-01T03:33:12.6459931Z 	from /var/lib/gems/2.7.0/gems/tractive-1.0.11/lib/tractive/revmap_generator.rb:29:in `block (2 levels) in generate'
2022-01-01T03:33:12.6461416Z 	from /var/lib/gems/2.7.0/gems/tractive-1.0.11/lib/tractive/revmap_generator.rb:23:in `foreach'
2022-01-01T03:33:12.6462844Z 	from /var/lib/gems/2.7.0/gems/tractive-1.0.11/lib/tractive/revmap_generator.rb:23:in `block in generate'
2022-01-01T03:33:12.6464271Z 	from /var/lib/gems/2.7.0/gems/tractive-1.0.11/lib/tractive/revmap_generator.rb:22:in `open'
2022-01-01T03:33:12.6465651Z 	from /var/lib/gems/2.7.0/gems/tractive-1.0.11/lib/tractive/revmap_generator.rb:22:in `generate'
2022-01-01T03:33:12.6467224Z 	from /var/lib/gems/2.7.0/gems/tractive-1.0.11/exe/generate.rb:27:in `revmap'
2022-01-01T03:33:12.6468397Z 	from /var/lib/gems/2.7.0/gems/thor-1.1.0/lib/thor/command.rb:27:in `run'
2022-01-01T03:33:12.6469579Z 	from /var/lib/gems/2.7.0/gems/thor-1.1.0/lib/thor/invocation.rb:127:in `invoke_command'
2022-01-01T03:33:12.6470732Z 	from /var/lib/gems/2.7.0/gems/thor-1.1.0/lib/thor.rb:392:in `dispatch'
2022-01-01T03:33:12.6471870Z 	from /var/lib/gems/2.7.0/gems/thor-1.1.0/lib/thor/invocation.rb:116:in `invoke'
2022-01-01T03:33:12.6473031Z 	from /var/lib/gems/2.7.0/gems/thor-1.1.0/lib/thor.rb:243:in `block in subcommand'
2022-01-01T03:33:12.6474155Z 	from /var/lib/gems/2.7.0/gems/thor-1.1.0/lib/thor/command.rb:27:in `run'
2022-01-01T03:33:12.6475327Z 	from /var/lib/gems/2.7.0/gems/thor-1.1.0/lib/thor/invocation.rb:127:in `invoke_command'
2022-01-01T03:33:12.6476483Z 	from /var/lib/gems/2.7.0/gems/thor-1.1.0/lib/thor.rb:392:in `dispatch'
2022-01-01T03:33:12.6477532Z 	from /var/lib/gems/2.7.0/gems/thor-1.1.0/lib/thor/base.rb:485:in `start'
2022-01-01T03:33:12.6478706Z 	from /var/lib/gems/2.7.0/gems/tractive-1.0.11/exe/tractive:80:in `<top (required)>'
2022-01-01T03:33:12.6479714Z 	from /usr/local/bin/tractive:23:in `load'
2022-01-01T03:33:12.6480556Z 	from /usr/local/bin/tractive:23:in `<main>'
2022-01-01T03:33:12.6576166Z Progress: [=============================                     ] 59.41% |
2022-01-01T03:33:12.6584956Z ##[error]Process completed with exit code 1.
2022-01-01T03:33:12.6760466Z Post job cleanup.
```

--- a/.github/workflows/convert.yml
+++ b/.github/workflows/convert.yml
@@ -27,6 +27,20 @@ jobs:
           - xml2rfc-bibxml
           - xml2rfc-website
 
+    # Service containers to run with `runner-job`
+    services:
+      # Label used to access the service container
+      svn:
+        # Docker Hub image
+        image: elleflorio/svn-server
+        volumes:
+          - ${GITHUB_WORKSPACE}/svn-server:/home/svn
+        ports:
+          # Maps tcp ports on service container to the host
+          - 80:80
+          - 443:443
+          - 3690:3690
+
     steps:
       - name: Setup Git
         run: |
@@ -70,6 +84,11 @@ jobs:
           repository: ietf-ribose/trac-svn-db
           token: ${{ secrets.IETF_RIBOSE_CI_PAT }}
           path: trac-svn-db
+
+      - name: Copy SVN server data to svn-server folder
+        run: |
+          ln -s trac-svn-db/svn/${{ matrix.svn_repo }} svn-server
+
 
       - name: Convert SVN to Git via reposurgeon
         if: ${{ matrix.svn_repo != 'xml2rfc-bibxml' && matrix.svn_repo != 'xml2rfc-website' && matrix.svn_repo != 'vocabulary_design_team_2013_2017' && matrix.svn_repo != 'datatracker' }}
@@ -128,47 +147,7 @@ jobs:
         working-directory: svn-git-maps
         run: |
           tractive generate revmap \
-            --svn-url https://svn.ietf.org/svn/tools/${{ matrix.svn_repo }} \
-            --git-local-repo-path ${GITHUB_WORKSPACE}/git/${{ matrix.svn_repo }} \
-            --rev-timestamp-file ${{ matrix.svn_repo }}.fo \
-            --revmap-output-file ./${{ matrix.svn_repo }}-revmap.txt
-
-      - name: Convert ${{ matrix.svn_repo }}.fo to ${{ matrix.svn_repo }}-revmap.txt
-        if: ${{ matrix.svn_repo == 'xml2rfc-bibxml' }}
-        working-directory: svn-git-maps
-        run: |
-          tractive generate revmap \
-            --svn-url https://svn.ietf.org/svn/tools/xml2rfc/website/rfcs \
-            --git-local-repo-path ${GITHUB_WORKSPACE}/git/${{ matrix.svn_repo }} \
-            --rev-timestamp-file ${{ matrix.svn_repo }}.fo \
-            --revmap-output-file ./${{ matrix.svn_repo }}-revmap.txt
-
-      - name: Convert ${{ matrix.svn_repo }}.fo to ${{ matrix.svn_repo }}-revmap.txt
-        if: ${{ matrix.svn_repo == 'xml2rfc-website' }}
-        working-directory: svn-git-maps
-        run: |
-          tractive generate revmap \
-            --svn-url https://svn.ietf.org/svn/tools/xml2rfc/website \
-            --git-local-repo-path ${GITHUB_WORKSPACE}/git/${{ matrix.svn_repo }} \
-            --rev-timestamp-file ${{ matrix.svn_repo }}.fo \
-            --revmap-output-file ./${{ matrix.svn_repo }}-revmap.txt
-
-      - name: Convert ${{ matrix.svn_repo }}.fo to ${{ matrix.svn_repo }}-revmap.txt
-        if: ${{ matrix.svn_repo == 'vocabulary_design_team_2013_2017' }}
-        working-directory: svn-git-maps
-        run: |
-          tractive generate revmap \
-            --svn-url https://svn.ietf.org/svn/tools/xml2rfc/vocabulary \
-            --git-local-repo-path ${GITHUB_WORKSPACE}/git/${{ matrix.svn_repo }} \
-            --rev-timestamp-file ${{ matrix.svn_repo }}.fo \
-            --revmap-output-file ./${{ matrix.svn_repo }}-revmap.txt
-
-      - name: Convert ${{ matrix.svn_repo }}.fo to ${{ matrix.svn_repo }}-revmap.txt
-        if: ${{ matrix.svn_repo == 'datatracker' }}
-        working-directory: svn-git-maps
-        run: |
-          tractive generate revmap \
-            --svn-url https://svn.ietf.org/svn/tools/ietfdb \
+            --svn-url svn:3690 \
             --git-local-repo-path ${GITHUB_WORKSPACE}/git/${{ matrix.svn_repo }} \
             --rev-timestamp-file ${{ matrix.svn_repo }}.fo \
             --revmap-output-file ./${{ matrix.svn_repo }}-revmap.txt
@@ -217,47 +196,7 @@ jobs:
         working-directory: svn-git-maps
         run: |
           tractive generate revmap \
-            --svn-url https://svn.ietf.org/svn/tools/${{ matrix.svn_repo }} \
-            --git-local-repo-path ${GITHUB_WORKSPACE}/git/${{ matrix.svn_repo }} \
-            --rev-timestamp-file ${{ matrix.svn_repo }}.fo \
-            --revmap-output-file ./${{ matrix.svn_repo }}-revmap.txt
-
-      - name: Update ${{ matrix.svn_repo }}-revmap.txt
-        if: ${{ matrix.svn_repo == 'xml2rfc-bibxml' }}
-        working-directory: svn-git-maps
-        run: |
-          tractive generate revmap \
-            --svn-url https://svn.ietf.org/svn/tools/xml2rfc/website/rfcs \
-            --git-local-repo-path ${GITHUB_WORKSPACE}/git/${{ matrix.svn_repo }} \
-            --rev-timestamp-file ${{ matrix.svn_repo }}.fo \
-            --revmap-output-file ./${{ matrix.svn_repo }}-revmap.txt
-
-      - name: Update ${{ matrix.svn_repo }}-revmap.txt
-        if: ${{ matrix.svn_repo == 'xml2rfc-website' }}
-        working-directory: svn-git-maps
-        run: |
-          tractive generate revmap \
-            --svn-url https://svn.ietf.org/svn/tools/xml2rfc/website \
-            --git-local-repo-path ${GITHUB_WORKSPACE}/git/${{ matrix.svn_repo }} \
-            --rev-timestamp-file ${{ matrix.svn_repo }}.fo \
-            --revmap-output-file ./${{ matrix.svn_repo }}-revmap.txt
-
-      - name: Update ${{ matrix.svn_repo }}-revmap.txt
-        if: ${{ matrix.svn_repo == 'vocabulary_design_team_2013_2017' }}
-        working-directory: svn-git-maps
-        run: |
-          tractive generate revmap \
-            --svn-url https://svn.ietf.org/svn/tools/xml2rfc/vocabulary \
-            --git-local-repo-path ${GITHUB_WORKSPACE}/git/${{ matrix.svn_repo }} \
-            --rev-timestamp-file ${{ matrix.svn_repo }}.fo \
-            --revmap-output-file ./${{ matrix.svn_repo }}-revmap.txt
-
-      - name: Update ${{ matrix.svn_repo }}-revmap.txt
-        if: ${{ matrix.svn_repo == 'datatracker' }}
-        working-directory: svn-git-maps
-        run: |
-          tractive generate revmap \
-            --svn-url https://svn.ietf.org/svn/tools/ietfdb \
+            --svn-url svn:3690 \
             --git-local-repo-path ${GITHUB_WORKSPACE}/git/${{ matrix.svn_repo }} \
             --rev-timestamp-file ${{ matrix.svn_repo }}.fo \
             --revmap-output-file ./${{ matrix.svn_repo }}-revmap.txt

--- a/.github/workflows/convert.yml
+++ b/.github/workflows/convert.yml
@@ -2,7 +2,7 @@ name: convert
 
 on:
   push:
-    branches: [ main, using_local_svn_server ]
+    branches: [ main ]
     paths-ignore:
       - '.github/workflows/install_reposurgeon.yml'
   schedule:

--- a/.github/workflows/convert.yml
+++ b/.github/workflows/convert.yml
@@ -85,11 +85,6 @@ jobs:
           token: ${{ secrets.IETF_RIBOSE_CI_PAT }}
           path: trac-svn-db
 
-      - name: Copy SVN server data to svn-server folder
-        run: |
-          ln -s trac-svn-db/svn/${{ matrix.svn_repo }} svn-server
-
-
       - name: Convert SVN to Git via reposurgeon
         if: ${{ matrix.svn_repo != 'xml2rfc-bibxml' && matrix.svn_repo != 'xml2rfc-website' && matrix.svn_repo != 'vocabulary_design_team_2013_2017' && matrix.svn_repo != 'datatracker' }}
         working-directory: conversion/${{ matrix.svn_repo }}
@@ -97,6 +92,7 @@ jobs:
           TERM: xterm-256color
         run: |
           ln -s ../../trac-svn-db/svn/${{ matrix.svn_repo }} ${{ matrix.svn_repo }}-mirror
+          ln -s ../../trac-svn-db/svn/${{ matrix.svn_repo }} ../../svn-server
           make
           mv ${{ matrix.svn_repo }}-git ../../git/${{ matrix.svn_repo }}
 
@@ -107,6 +103,7 @@ jobs:
           TERM: xterm-256color
         run: |
           ln -s ../../trac-svn-db/svn/xml2rfc ${{ matrix.svn_repo }}-mirror
+          ln -s ../../trac-svn-db/svn/xml2rfc ../../svn-server
           make
           mv ${{ matrix.svn_repo }}-git ../../git/${{ matrix.svn_repo }}
 
@@ -117,6 +114,7 @@ jobs:
           TERM: xterm-256color
         run: |
           ln -s ../../trac-svn-db/svn/ietfdb ${{ matrix.svn_repo }}-mirror
+          ln -s ../../trac-svn-db/svn/ietfdb ../../svn-server
           make
           mv ${{ matrix.svn_repo }}-git ../../git/${{ matrix.svn_repo }}
 

--- a/.github/workflows/convert.yml
+++ b/.github/workflows/convert.yml
@@ -92,7 +92,7 @@ jobs:
           TERM: xterm-256color
         run: |
           ln -s ../../trac-svn-db/svn/${{ matrix.svn_repo }} ${{ matrix.svn_repo }}-mirror
-          cp -r ../../trac-svn-db/svn/${{ matrix.svn_repo }} ../../svn-server
+          cp -r ../../trac-svn-db/svn/${{ matrix.svn_repo }}/. ../../svn-server
           make
           mv ${{ matrix.svn_repo }}-git ../../git/${{ matrix.svn_repo }}
 
@@ -103,7 +103,7 @@ jobs:
           TERM: xterm-256color
         run: |
           ln -s ../../trac-svn-db/svn/xml2rfc ${{ matrix.svn_repo }}-mirror
-          cp -r ../../trac-svn-db/svn/xml2rfc ../../svn-server
+          cp -r ../../trac-svn-db/svn/xml2rfc/. ../../svn-server
           make
           mv ${{ matrix.svn_repo }}-git ../../git/${{ matrix.svn_repo }}
 
@@ -114,7 +114,7 @@ jobs:
           TERM: xterm-256color
         run: |
           ln -s ../../trac-svn-db/svn/ietfdb ${{ matrix.svn_repo }}-mirror
-          cp -r ../../trac-svn-db/svn/ietfdb ../../svn-server
+          cp -r ../../trac-svn-db/svn/ietfdb/. ../../svn-server
           make
           mv ${{ matrix.svn_repo }}-git ../../git/${{ matrix.svn_repo }}
 

--- a/.github/workflows/convert.yml
+++ b/.github/workflows/convert.yml
@@ -92,7 +92,7 @@ jobs:
           TERM: xterm-256color
         run: |
           ln -s ../../trac-svn-db/svn/${{ matrix.svn_repo }} ${{ matrix.svn_repo }}-mirror
-          cp ../../trac-svn-db/svn/${{ matrix.svn_repo }} ../../svn-server
+          cp -r ../../trac-svn-db/svn/${{ matrix.svn_repo }} ../../svn-server
           make
           mv ${{ matrix.svn_repo }}-git ../../git/${{ matrix.svn_repo }}
 
@@ -103,7 +103,7 @@ jobs:
           TERM: xterm-256color
         run: |
           ln -s ../../trac-svn-db/svn/xml2rfc ${{ matrix.svn_repo }}-mirror
-          cp ../../trac-svn-db/svn/xml2rfc ../../svn-server
+          cp -r ../../trac-svn-db/svn/xml2rfc ../../svn-server
           make
           mv ${{ matrix.svn_repo }}-git ../../git/${{ matrix.svn_repo }}
 
@@ -114,7 +114,7 @@ jobs:
           TERM: xterm-256color
         run: |
           ln -s ../../trac-svn-db/svn/ietfdb ${{ matrix.svn_repo }}-mirror
-          cp ../../trac-svn-db/svn/ietfdb ../../svn-server
+          cp -r ../../trac-svn-db/svn/ietfdb ../../svn-server
           make
           mv ${{ matrix.svn_repo }}-git ../../git/${{ matrix.svn_repo }}
 

--- a/.github/workflows/convert.yml
+++ b/.github/workflows/convert.yml
@@ -92,7 +92,7 @@ jobs:
           TERM: xterm-256color
         run: |
           ln -s ../../trac-svn-db/svn/${{ matrix.svn_repo }} ${{ matrix.svn_repo }}-mirror
-          cp -r ../../trac-svn-db/svn/${{ matrix.svn_repo }}/. ../../svn-server
+          cp -rT ../../trac-svn-db/svn/${{ matrix.svn_repo }} ../../svn-server
           make
           mv ${{ matrix.svn_repo }}-git ../../git/${{ matrix.svn_repo }}
 
@@ -103,7 +103,7 @@ jobs:
           TERM: xterm-256color
         run: |
           ln -s ../../trac-svn-db/svn/xml2rfc ${{ matrix.svn_repo }}-mirror
-          cp -r ../../trac-svn-db/svn/xml2rfc/. ../../svn-server
+          cp -rT ../../trac-svn-db/svn/xml2rfc/. ../../svn-server
           make
           mv ${{ matrix.svn_repo }}-git ../../git/${{ matrix.svn_repo }}
 
@@ -114,7 +114,7 @@ jobs:
           TERM: xterm-256color
         run: |
           ln -s ../../trac-svn-db/svn/ietfdb ${{ matrix.svn_repo }}-mirror
-          cp -r ../../trac-svn-db/svn/ietfdb/. ../../svn-server
+          cp -rT ../../trac-svn-db/svn/ietfdb/. ../../svn-server
           make
           mv ${{ matrix.svn_repo }}-git ../../git/${{ matrix.svn_repo }}
 

--- a/.github/workflows/convert.yml
+++ b/.github/workflows/convert.yml
@@ -103,7 +103,7 @@ jobs:
           TERM: xterm-256color
         run: |
           ln -s ../../trac-svn-db/svn/xml2rfc ${{ matrix.svn_repo }}-mirror
-          sudo cp -rT ../../trac-svn-db/svn/xml2rfc/. ../../svn-server
+          sudo cp -rT ../../trac-svn-db/svn/xml2rfc ../../svn-server
           make
           mv ${{ matrix.svn_repo }}-git ../../git/${{ matrix.svn_repo }}
 
@@ -114,7 +114,7 @@ jobs:
           TERM: xterm-256color
         run: |
           ln -s ../../trac-svn-db/svn/ietfdb ${{ matrix.svn_repo }}-mirror
-          sudo cp -rT ../../trac-svn-db/svn/ietfdb/. ../../svn-server
+          sudo cp -rT ../../trac-svn-db/svn/ietfdb ../../svn-server
           make
           mv ${{ matrix.svn_repo }}-git ../../git/${{ matrix.svn_repo }}
 
@@ -141,7 +141,6 @@ jobs:
           cp ${GITHUB_WORKSPACE}/conversion/${{ matrix.svn_repo }}/${{ matrix.svn_repo }}.fo .
 
       - name: Convert ${{ matrix.svn_repo }}.fo to ${{ matrix.svn_repo }}-revmap.txt
-        if: ${{ matrix.svn_repo != 'xml2rfc-bibxml' && matrix.svn_repo != 'xml2rfc-website' && matrix.svn_repo != 'vocabulary_design_team_2013_2017' && matrix.svn_repo != 'datatracker' }}
         working-directory: svn-git-maps
         run: |
           tractive generate revmap \
@@ -190,7 +189,6 @@ jobs:
             return new_message'
 
       - name: Update ${{ matrix.svn_repo }}-revmap.txt
-        if: ${{ matrix.svn_repo != 'xml2rfc-bibxml' && matrix.svn_repo != 'xml2rfc-website' && matrix.svn_repo != 'vocabulary_design_team_2013_2017' && matrix.svn_repo != 'datatracker' }}
         working-directory: svn-git-maps
         run: |
           tractive generate revmap \

--- a/.github/workflows/convert.yml
+++ b/.github/workflows/convert.yml
@@ -2,7 +2,7 @@ name: convert
 
 on:
   push:
-    branches: [ main ]
+    branches: [ main, using_local_svn_server ]
     paths-ignore:
       - '.github/workflows/install_reposurgeon.yml'
   schedule:

--- a/.github/workflows/convert.yml
+++ b/.github/workflows/convert.yml
@@ -92,7 +92,7 @@ jobs:
           TERM: xterm-256color
         run: |
           ln -s ../../trac-svn-db/svn/${{ matrix.svn_repo }} ${{ matrix.svn_repo }}-mirror
-          ln -s ../../trac-svn-db/svn/${{ matrix.svn_repo }} ../../svn-server
+          cp ../../trac-svn-db/svn/${{ matrix.svn_repo }} ../../svn-server
           make
           mv ${{ matrix.svn_repo }}-git ../../git/${{ matrix.svn_repo }}
 
@@ -103,7 +103,7 @@ jobs:
           TERM: xterm-256color
         run: |
           ln -s ../../trac-svn-db/svn/xml2rfc ${{ matrix.svn_repo }}-mirror
-          ln -s ../../trac-svn-db/svn/xml2rfc ../../svn-server
+          cp ../../trac-svn-db/svn/xml2rfc ../../svn-server
           make
           mv ${{ matrix.svn_repo }}-git ../../git/${{ matrix.svn_repo }}
 
@@ -114,7 +114,7 @@ jobs:
           TERM: xterm-256color
         run: |
           ln -s ../../trac-svn-db/svn/ietfdb ${{ matrix.svn_repo }}-mirror
-          ln -s ../../trac-svn-db/svn/ietfdb ../../svn-server
+          cp ../../trac-svn-db/svn/ietfdb ../../svn-server
           make
           mv ${{ matrix.svn_repo }}-git ../../git/${{ matrix.svn_repo }}
 

--- a/.github/workflows/convert.yml
+++ b/.github/workflows/convert.yml
@@ -194,7 +194,7 @@ jobs:
         working-directory: svn-git-maps
         run: |
           tractive generate revmap \
-            --svn-url svn:3690 \
+            --svn-url svn://localhost:3690 \
             --git-local-repo-path ${GITHUB_WORKSPACE}/git/${{ matrix.svn_repo }} \
             --rev-timestamp-file ${{ matrix.svn_repo }}.fo \
             --revmap-output-file ./${{ matrix.svn_repo }}-revmap.txt

--- a/.github/workflows/convert.yml
+++ b/.github/workflows/convert.yml
@@ -145,7 +145,7 @@ jobs:
         working-directory: svn-git-maps
         run: |
           tractive generate revmap \
-            --svn-url svn:3690 \
+            --svn-url svn://localhost:3690 \
             --git-local-repo-path ${GITHUB_WORKSPACE}/git/${{ matrix.svn_repo }} \
             --rev-timestamp-file ${{ matrix.svn_repo }}.fo \
             --revmap-output-file ./${{ matrix.svn_repo }}-revmap.txt

--- a/.github/workflows/convert.yml
+++ b/.github/workflows/convert.yml
@@ -92,7 +92,7 @@ jobs:
           TERM: xterm-256color
         run: |
           ln -s ../../trac-svn-db/svn/${{ matrix.svn_repo }} ${{ matrix.svn_repo }}-mirror
-          cp -rT ../../trac-svn-db/svn/${{ matrix.svn_repo }} ../../svn-server
+          sudo cp -rT ../../trac-svn-db/svn/${{ matrix.svn_repo }} ../../svn-server
           make
           mv ${{ matrix.svn_repo }}-git ../../git/${{ matrix.svn_repo }}
 
@@ -103,7 +103,7 @@ jobs:
           TERM: xterm-256color
         run: |
           ln -s ../../trac-svn-db/svn/xml2rfc ${{ matrix.svn_repo }}-mirror
-          cp -rT ../../trac-svn-db/svn/xml2rfc/. ../../svn-server
+          sudo cp -rT ../../trac-svn-db/svn/xml2rfc/. ../../svn-server
           make
           mv ${{ matrix.svn_repo }}-git ../../git/${{ matrix.svn_repo }}
 
@@ -114,7 +114,7 @@ jobs:
           TERM: xterm-256color
         run: |
           ln -s ../../trac-svn-db/svn/ietfdb ${{ matrix.svn_repo }}-mirror
-          cp -rT ../../trac-svn-db/svn/ietfdb/. ../../svn-server
+          sudo cp -rT ../../trac-svn-db/svn/ietfdb/. ../../svn-server
           make
           mv ${{ matrix.svn_repo }}-git ../../git/${{ matrix.svn_repo }}
 

--- a/.github/workflows/convert.yml
+++ b/.github/workflows/convert.yml
@@ -34,7 +34,7 @@ jobs:
         # Docker Hub image
         image: elleflorio/svn-server
         volumes:
-          - ${GITHUB_WORKSPACE}/svn-server:/home/svn
+          - ${{ github.workspace }}/svn-server:/home/svn
         ports:
           # Maps tcp ports on service container to the host
           - 80:80

